Makefile

```diff
@@ -2,52 +2,33 @@
 
 all: deps compile
 
-compile: rebar
-	./rebar compile
+compile: rebar3
+	./rebar3 compile
 
-deps: rebar
-	./rebar get-deps
+deps: rebar3
+	./rebar3 get-deps
 
-clean: rebar
-	./rebar clean
+clean: rebar3
+	./rebar3 clean
 
-test-deps: rebar
-	./rebar -C rebar.test.config get-deps
+test-deps: rebar3
+	./rebar3 get-deps
 
-test-compile: rebar test-deps
-	./rebar -C rebar.test.config compile
+test-compile: rebar3 test-deps
+	./rebar3 compile
 
 test: test-compile
-	./rebar -C rebar.test.config skip_deps=true eunit
+	./rebar3 eunit
 
-coverage-report: $(shell ls -1rt `find .eunit -type f -name \*.coverdata 2>/dev/null` | tail -n1)
-	erl -noshell -pa ebin deps/*/ebin -eval 'ecoveralls:travis_ci("$?"), init:stop()'
+coverage-report: _build/test/cover/eunit.coverdata
+	./rebar3 coveralls send
 
-rebar:
-	wget https://github.com/rebar/rebar/releases/download/2.5.1/rebar &&\
-	chmod u+x rebar
+codecov: rebar3 test
```

Comment:
The test `target` will be executed twice now, see https://travis-ci.org/esl/exml/jobs/426567957#L589 and https://travis-ci.org/esl/exml/jobs/426567957#L653

I think it's enough set the same dependencies for the `codecov` target as for `coverage-report`.